Header header
Person[] persons
uint32 image_w
uint32 image_h

uint32 crop_w
uint32 crop_h
uint32 crop_x
uint32 crop_y